Makefile

```
@@ -134,6 +134,11 @@ deploy-community-policy-framework-managed: deploy-policy-framework-managed-crd-o
 kind-deploy-policy-framework:
 	@echo installing policy-propagator on hub
 	kubectl create ns $(KIND_HUB_NAMESPACE) --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
+	@echo "wait until pods are up"
+	kubectl wait deployment -n cert-manager cert-manager --for condition=Available=True --timeout=180s
+	kubectl wait --for=condition=Ready pod -l app.kubernetes.io/instance=cert-manager -n cert-manager --timeout=180s 
+	sed 's/namespace: open-cluster-management/namespace: $(KIND_HUB_NAMESPACE)/g'  https://raw.githubusercontent.com/stolostron/governance-policy-propagator/main/deploy/webhook.yaml | kubectl apply -f-
```

This is tricky because we need to have this pull from the correct branch. Maybe let's ignore the error if the file isn't there? (i.e. the leading hyphen)
```suggestion
	-sed 's/namespace: open-cluster-management/namespace: $(KIND_HUB_NAMESPACE)/g'  https://raw.githubusercontent.com/stolostron/governance-policy-propagator/${RELEASE_BRANCH}/deploy/webhook.yaml | kubectl apply -f-
```

---

good point !!!